invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/subjects.html

```diff
@@ -22,7 +22,7 @@
     </div>
   </div>
   {%- endfor %}
-  {% set keywords = subjects|rejectattr("scheme") %}
+  {% set keywords = subjects|rejectattr("scheme")|list %}
```

Comment:
Huh... what was it returning before?

Reply:
IIRC it was a "select_or_reject" object.

Reply:
ha, super strange thanks!